db/seeds.rb

```diff
@@ -1,3 +1,4 @@
+Book.delete_all
```

Comment:
Interesting. I don't have any dependencies set up at the moment, but it's good to know what the difference is. I've been kind of wondering this over the past couple days. Thanks for the link.